code/src/java/pcgen/cdom/base/Loadable.java

```diff
@@ -19,17 +19,65 @@
 
 import java.net.URI;
 
+/**
+ * A Loadable is an object that PCGen can load from its persistent file storage (generally "LST" files). 
+ */
 public interface Loadable extends Identified
 {
 
+	/**
+	 * Sets the name of the Loadable to the given name.
+	 * 
+	 * @param name
+	 *            The name to be given to the Loadable
+	 */
 	public void setName(String name);
 
+	/**
+	 * Returns the source URI for this Loadable.
+	 * 
+	 * @return The source URI for this Loadable
+	 */
 	public URI getSourceURI();
 
+	/**
+	 * Sets the source URI for this Loadable.
+	 * 
+	 * @param source
+	 *            The source URI for this Loadable
+	 */
 	public void setSourceURI(URI source);
 
+	/**
+	 * Returns true if this object is "internal" (meaning PCGen creates it and it doesn't
+	 * need to be persisted to a set of saved LST data).
+	 * 
+	 * @return true if this object is "internal"; false otherwise
+	 */
 	public boolean isInternal();
 
+	/**
+	 * Returns true if the object is of the given type.
+	 * 
+	 * Note that the given String may contain a TYPE= or TYPE. prefix (which will be
+	 * discarded) and can contain more than one type (separated by periods '.'). If more
+	 * than one type is provided, this Loadable must be of ALL of the given types for this
+	 * method to return true.
+	 * 
+	 * @param type
+	 *            The type to be checked to see if this Loadable is of the given Type
+	 * @return true if the object is of the given type; false otherwise.
+	 */
 	public boolean isType(String type);
```

Comment:
Since this is not creating this method, but is simply documenting the method, I don't think a cross-cutting refactor that will hit another ~50 classes should be done as part of one PR.